packages/wrangler/src/type-generation/index.ts

```diff
@@ -63,7 +63,7 @@ export async function typesHandler(
 
 	await printWranglerBanner();
 
-	const configPath = resolveWranglerConfigPath(args);
+	const configPath = resolveWranglerConfigPath(args, {});
```

Comment:
Should type generation not use this? What if the generated config file has a different set of compat flags to the user one?

Reply:
Yes I thought about that. In that case the code that expects to have the compat flag is part of the generated code, not the user code. If the user wants to use such APIs then they should be defining the compat flag in their user config.